packages/mask/src/manifest.json

```diff
@@ -10,6 +10,6 @@
     "browser_action": { "default_popup": "popups.html" },
     "homepage_url": "https://mask.io",
     "description": "The portal to the new & open Internet. Send encrypted message and decentralized Apps right on top of social networks.",
-    "web_accessible_resources": ["js/*", "*.svg", "*.png", "*.css"],
+    "web_accessible_resources": ["js/*", "*.svg", "*.png", "*.css", "https://assets.coingecko.com/*"],
```

Comment:
I don't know it works! 🤔

Reply:
If this declaration is reliable. Why not add more asset urls?